.github/workflows/build-lint-test.yml

```diff
@@ -95,12 +95,16 @@ jobs:
   policy:
     name: Generate LavaMoat policy
     runs-on: ubuntu-latest
-    needs: prepare
+    needs: build
     steps:
       - name: Checkout and setup environment
         uses: MetaMask/action-checkout-and-setup@v1
         with:
           is-high-risk-environment: false
+      - name: Restore build files
```

Comment:
Why do we need build files for this?

Reply:
Because this uses the production build, which doesn't use `tsconfig` paths.